src/plugins/visualizations/public/wizard/new_vis_modal.tsx

```diff
@@ -143,10 +143,12 @@ class NewVisModal extends React.Component<TypeSelectionProps, TypeSelectionState
     }
 
     let params;
-    if ('aliasPath' in visType) {
-      params = visType.aliasPath;
-      this.props.onClose();
-      this.navigate(visType.aliasApp, visType.aliasPath);
+    if ('alias' in visType) {
+      if ('path' in visType.alias) {
+        params = visType.alias.path;
+        this.props.onClose();
+        this.navigate(visType.alias.app, visType.alias.path);
+      }
```

Comment:
In the future, we could extend this to allow creation through the wizard via the embeddable factory; however, at least for the Link embeddable, I don't see much value being added by allowing inline/flyout visualization creation outside of the Dashboard app.